.config/forest.dic

```diff
@@ -27,6 +27,7 @@ crypto
 daemon
 daemonize
 DB/S
+deserialized
```

Comment:
we won't have to add `deserialize` next time (I think [it](https://github.com/drahnr/cargo-spellcheck/blob/dff48db8ca954fce14a0cd5aea127ce59a929624/hunspell-data/en_US.aff#L67-L71) should work)
```suggestion
deserialize/D
```